hack/lib/strimzi.bash

```diff
@@ -138,6 +138,11 @@ spec:
           name: "*"
         operation: Read
         host: "*"
+      - resource:
```

Comment:
which script? 

This is granting permissions to delete consumer groups metadata which is (and actually already was) needed to delete consumer groups.

Reply:
I more meant: was this temporary, if not, I wonder why it was not there before.

script, well: should we do this permanent in this `.bash` file/scrip, looks like yes


and upstream, we may want this too? I see only topic delete on the "manifests" there

Reply:
> and upstream, we may want this too? I see only topic delete on the "manifests" there

yes, definitely

Reply:
>  I wonder why it was not there before.

it's because we gave up on consumer group deletion after some failures, currently, we don't with the new source v2

Reply:
ok, thanks!